cmd/rac/output.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+func info() {
+	println(`
+### CSVs ###
+
+The first row:
+- "CODE": Means the following column says what version of the code produced
+  the output.
+- "v0.2.2": The code version
+- "RAMES": Means the following column says what RAMSES specification was used.
+- "SPU045-S2:6F": The RAMSES version
+- "INNOSAT": Means the following column says what INNOSAT specification was
+  used.
+- "IS-OSE-ICD-0005:1": The INNOSAT version
+- "AEZ": Means the following column says what AEZ specification was used.
+- "AEZICD002:H": The AEZ version
+
+The header row starts with a couple of columns common to all output and then
+follows columns specific to each file
+
+
+- File
+  The full path to the rac-file on the computer that produced the csv
+- ProcessingDate
+  The local time when the file was processed
+- RamsesTime (Ramses Header)
+  The time when the ramses file was created (UTC)
+
+- QualityIndicator (Ramses TM Header)
+  Indicates whether the transported data is complete or partial
+  (0 = Complete, 1 = partial).
+- LossFlag (Ramses TM Header)
+  Used to indicate that a sequence discontinuity has been detected
+- VCFrameCounter (Ramses TM Header)
+  Counter of the transfer frame the payload packet arrived in.
+
+- SPSequenceCount (Innosat Source Header)

```

Comment:
No description?

Reply:
duplicated comment?

Reply:
Yes. No description?

Reply:
Added suggestion, but that spec is so confusing to read to just wanted to check with joakim before committing it.